### ISSUE TITLE
Output for Test Dataset

### ISSUE THREAD

**Issue by raufs** [ohlab/SMEG#10]
Hi,

Thank you for developing such a really great method and very cool study too! 

I am wondering if it is possible to share the expected results of the test dataset provided to help check if we have SMEG set up correctly? 

Kind regards,
Rauf

**Comment by aemiol** [comment]
Hi, 
Sorry for the slow response. Below is the expected result for the test set

Cluster SMEG    Coverage        No of SNPs      SMEG range
cluster1        1.75370127210152        4.08426832179518        10971   1.683 - 1.808
cluster2        1.12409093832417        2.08503190107843        9164    1.114 - 1.169

Thanks.
Tunde

**Comment by raufs** [comment]
Thank you Tunde!

It appears I see similar trends in the results but differences in the values beyond simple rounding error. For instance, I also get that cluster1 has a higher SMEG value than cluster2. I ended up installing through conda and manual tinkering of certain steps in the pipeline, so perhaps I broke something in the process. 

Also, I saw it was recommended that we use the Singularity instance, however, it seems that the instance provided might not be compatible with the new Singularity version (at least that's what I interpreted the issue as). Would it be possible to update the Singularity instance/container to work with the latest version? 

My exact results are the following:

Cluster	SMEG	Coverage	No of SNPs	SMEG range
cluster1	1.49942916402499	1.21812828266752	5090	1.416 - 1.517
cluster2	1.17742009741453	2.70058155948281	11114	1.139 - 1.199

Kind regards,
Rauf
